roadmap.md

```diff
@@ -25,49 +18,87 @@ state.
 
-   Enhancement: was https://github.com/kubernetes/enhancements/issues/3027
+   Enhancement: _none_
 
-   Project board: https://github.com/orgs/kubernetes/projects/171
+   Project board: https://github.com/orgs/kubernetes/projects/171/views/1
 
-1. **Making deb and rpm package builds more robust**
+1. **Rework supported platforms**
 
-   Outcome: Provide a fast and error-resistant workflow for building and
-   publishing packages to the community-owned infrastructure.
+   Outcome: Which platforms Kubernetes supports reflected in documentation and tooling.
 
-   Enhancement: https://github.com/kubernetes/enhancements/issues/1731
+   Enhancement: **TBD** ([Supported Platform Tiers](https://docs.google.com/document/d/1QnT2LJ1dNYFyvOCt_X6F8ry0KgAg8H2MjoZnt5Xgtog))
 
-   Project board: https://github.com/orgs/kubernetes/projects/137
+   Project board: _none_
 
-1. **Enable other Kubernetes subprojects to use our packages infrastructure**
+1. **Improve SIG Release contributor ladder & sustainability**
+   Outcome: Up-to-date documentation and project boards for new and long term contributors.
+   Enhancement: _none_
+   Project board: _none_
 
-   Outcome: Building a CLI, library and documentation for Kubernetes related
-   projects to use the existing packaging infrastructure.
+### Known Risks
 
-   Enhancement: _none_
+1. **We rely on different SIGs for our work**
 
-   Project board: https://github.com/orgs/kubernetes/projects/137
+   We have a need to discuss our enhancements with different SIGs to get all
+   required information and drive the change. This can lead into helpful, but
+   maybe not expected input and delay the deliverable.
 
-1. **Migrate away from GCS bucket: gs://kubernetes-release**
+1. **Some topics require initial research**
 
-   Outcome: Ensure we migrate away from gs://kubernetes-release, use a
-   community-owned Object Storage bucket for binaries releases, serve the
-   binaries published through a CDN service.
+   We're not completely aware of all technical aspects for the changes. This
+   means that there is a risk of delaying because of investing more time in
+   pre-research.
+
+### Requests to Other Teams
+
+1. **SIG K8s Infra**
+
+   For general infrastructure support we rely on.
+
+1. **SIG Architecture**
+
+   For the formalization of the released platforms and input about the overall
+   supply chain.
+
+1. **SIG Security**
+
+   For content which should be part of the release metadata.
+
+1. **SIG Contribex**
+
+   For getting input about sustainable contributor experience.
+
+### Other Priorities
+
+This section covers items from the past 2022 and 2024 roadmap which have not been
+started or became stale.
+
+1. **Make artifact validation more robust**
+
+   Outcome: Provide a fast and error-resistant workflow for artifact promotion compliant with SLSA.
+
+   Enhancement: was kubernetes/enhancements#3027
+
+   Project board: https://github.com/orgs/kubernetes/projects/171
```

Comment:
Given we want to do a rewrite, do we think this board is still relevant ? some issues are about investigate _technical debt_ (for the lack of a better term)